app/models/talk.rb

```diff
@@ -95,4 +95,8 @@ def thumbnail_lg
   def thumbnail_xl
     self[:thumbnail_xl].presence || "https://i.ytimg.com/vi/#{video_id}/maxresdefault.jpg"
   end
+
+  def related_talks(limit: 6)
+    Talk.order("RANDOM()").excluding(self).limit(limit)
```

Comment:
this is not really "related" 😉

a youtube video has "tags". 

if you find talks with the highest amount of matching tags, that would really make the results more "related"

Reply:
Yes for sure this is preparation work for #19, the vector base search should give us more relevant recommendations once we got it in place. 

The main goal of this PR was to split in a frame this list of recommended talks so that the talks#show is less coupled to the recommendation engine. 

As for the youtube video tags unfortunately I don't think lots of the channels I scrapped had some defined. RailsWorlds has some in the description that would need to be parsed. 
I want to have down the road some tags for the talks so that we can have a much better facet search experience. I thought they could either be crowdsourced and user-generated our check if I could generate some using OpenAI and the video transcript